lib/ui/home/home_screen.dart

```diff
@@ -29,29 +60,52 @@ class _HomeScreenState extends ConsumerState<HomeScreen> {
   Widget build(BuildContext context) {
     ref.listen<HomeViewState>(homeViewModelProvider, (_, state) {
       state.maybeWhen(
-          getUserProfileSuccess: (profile) => {
-                ref.read(_profileAvatarProvider.notifier).state =
-                    profile.avatarUrl,
-              },
-          loading: () {
-            ref.read(_loadingStateProvider.notifier).state = true;
-          },
-          error: (message) => {
-                ref.read(_loadingStateProvider.notifier).state = false,
-                showToastMessage(message)
-              },
+          success: () => {},
+          loading: () {},
+          error: (message) => {showToastMessage(message)},
```

Comment:
```suggestion
          error: (message) => { showToastMessage(message) },
```

Reply:
@manh-t Not sure why but when I use auto-formatting, it removed the space between braces 😵‍💫

Reply:
@hoangnguyen92dn ah I see, we can remove the bracket
```suggestion
error: (message) => showToastMessage(message),
```